tck/features/expressions/path/Path3.feature

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright (c) 2015-2020 "Neo Technology,"
+# Network Engine for Objects in Lund AB [http://neotechnology.com]
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Attribution Notice under the terms of the Apache License 2.0
+#
+# This work was created by the collective efforts of the openCypher community.
+# Without limiting the terms of Section 6, any Derivative Work that is not
+# approved by the public consensus process of the openCypher Implementers Group
+# should not be described as “Cypher” (and Cypher® is a registered trademark of
+# Neo4j Inc.) or as "openCypher". Extensions by implementers or prototypes or
+# proposals for change that have been documented or implemented should only be
+# described as "implementation extensions to Cypher" or as "proposed changes to
+# Cypher that are not yet approved by the openCypher community".
+#
+
+#encoding: utf-8
+
+Feature: Path3 - Length of a path
+
+  Scenario: Pass the path length test
+    Given an empty graph
+    And having executed:
+      """
+      CREATE (a:A {name: 'A'})-[:KNOWS]->(b:B {name: 'B'})
+      """
+    When executing query:
+      """
+      MATCH p = (n)-->(x)
+      WHERE length(p) = 1
+      RETURN x
+      """
+    Then the result should be, in any order:
+      | x                |
+      | (:B {name: 'B'}) |
+    And no side effects
+
+  Scenario: Do not return anything because path length does not match
+    Given an empty graph
+    And having executed:
+      """
+      CREATE (a:A {name: 'A'})-[:KNOWS]->(b:B {name: 'B'})
+      """
+    When executing query:
+      """
+      MATCH p = (n)-->(x)
+      WHERE length(p) = 10
+      RETURN x
+      """
+    Then the result should be, in any order:
+      | x |
+    And no side effects
+
+  Scenario: Return a var length path of length zero
```

Comment:
I think this does not have to be duplicated. It is clearly testing the length function.

Hence, it should only be here (removed from its original feature) and be at the top of this feature file.